packages/elements/src/components/ino-card/readme.md

```diff
@@ -59,14 +59,6 @@ class MyComponent extends Component {
 | `inoSelected`         | `ino-selected`          | Selects the card and displays a check mark icon on top of the card | `boolean` | `false` |
 
 
-## CSS Custom Properties
```

Comment:
This seems to be happening while running the `yarn start` script. Run `yarn build` in `packages/elements` to restore this table

Reply:
Done ✅